cmd/state/state.go

```diff
@@ -52,7 +52,7 @@ type GlobalState struct {
 	SignalNotify func(chan<- os.Signal, ...os.Signal)
 	SignalStop   func(chan<- os.Signal)
 
-	Logger         *logrus.Logger
+	Logger         *logrus.Logger //nolint:forbidigo //TODO change in the future
```

Comment:
Let's be explicit about TODO; I guess you meant to change to the `logrus.FieldLogger` interface in that context, but it's not straightforward from the TODO.